### ISSUE TITLE
Yazi doesnt visually remove the deleted file.

### ISSUE THREAD

**Issue by I-Own-You** [mikavilpas/yazi.nvim#34]
Hi!

If you have, for example a file somewhere nested in folders, and you find it with `s` or `S`, and then remove it with `d` or `D`, the file is removed, but visually its still there. In order it to disappear, you need to go back one level, then it will show the folder without the file.

**Comment by mikavilpas** [comment]
Hi, I'm not sure if I got that correctly, but it sounds like this is a bug in yazi itself. I may have run into something like that but can't quite be sure.

Would you like to open a bug report to the yazi repository?

**Comment by I-Own-You** [comment]
> Hi, I'm not sure if I got that correctly, but it sounds like this is a bug in yazi itself. I may have run into something like that but can't quite be sure.
> 
> Would you like to open a bug report to the yazi repository?

yeah, seems its yazi itself, ill close this and report an issue there
